tests/end-to-end/upgrade/main.go

```diff
@@ -101,14 +98,11 @@ func main() {
 	messagingCli, err := messagingclientv1alpha1.NewForConfig(k8sConfig)
 	fatalOnError(err, "while creating knative Messaging clientset")
 
-	kubelessCli, err := kubeless.NewForConfig(k8sConfig)
-	fatalOnError(err, "while creating Kubeless clientset")
-
 	domainName, err := getDomainNameFromCluster(k8sCli)
 	fatalOnError(err, "while reading domain name from cluster")
 
-	kymaAPI, err := kyma.NewForConfig(k8sConfig)
-	fatalOnError(err, "while creating Kyma Api clientset")
+	//kymaAPI, err := kyma.NewForConfig(k8sConfig)
+	//fatalOnError(err, "while creating Kyma Api clientset")
```

Comment:
Should the comments be there? There are a few more in this file

Reply:
Good point!